src/app/models/project.ts

```diff
@@ -8,8 +8,7 @@ export enum ProjectVisibility {
     Public
 }
 
-export class Project {
-    id: string;
+export class ProjectData {
```

Comment:
Only have one model class per file. Split this file into two named `project-data.ts` and `api-project.ts`

This makes it easy to always fuzzy search to find the model files, rather than having to guess which file the model is defined in.